packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -128,8 +128,8 @@ export function MainContent() {
         />
       </SidePanel>
 
-      {/* Preferences panel */}
-      <PreferencesDialog />
+      {/* Settings panel */}
+      <SettingsDialog />
```

Comment:
NIT: This was earlier implemented as a Dialog/Modal but eventually turned into a Sidepanel. We can rename this to `SettingsPanel` as well and get rid of the Dialog wording.

Reply:
Not sure which part you mean, I think I've already renamed everything to `Settings..`?

Reply:
I meant renaming `SettingsDialog` component to `SettingsPanel` since it's not a dialog now and we've already updated it's name so it would not create any additional diffs.

Reply:
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/1109